index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
+          [[RDF11-CONCEPTS]].
+        </p>
+      </dd>
+
+      <!-- 
+      Security considerations shared by media types employing this
+      structured syntax must be specified here.  The same requirements
+      for media type security considerations given in Section 4.6 apply
+      here, with the exception that the option of not assessing the
+      security considerations is not available for suffix registrations.
+      -->
+      <dt>Security considerations</dt>
+      <dd>
+        See <a data-cite="RFC8259#section-12">RFC&nbsp;8259, section 12</a> [[RFC8259]]
+        <p>Since JSON-LD is intended to be a pure data exchange format for
+          directed graphs, the serialization SHOULD NOT be passed through a
+          code execution mechanism such as JavaScript's <code>eval()</code>
+          function to be parsed. An (invalid) document may contain code that,
+          when executed, could lead to unexpected side effects compromising
+          the security of a system.</p>
+        <p>When processing JSON-LD documents, links to remote contexts and frames are
+          typically followed automatically, resulting in the transfer of files
+          without the explicit request of the user for each one. If remote
+          contexts are served by third parties, it may allow them to gather
+          usage patterns or similar information leading to privacy concerns.
+          Specific implementations, such as the API defined in the
+          JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]],
+          may provide fine-grained mechanisms to control this behavior.</p>
+        <p>JSON-LD contexts that are loaded from the Web over non-secure connections,
+          such as HTTP, run the risk of being altered by an attacker such that
+          they may modify the JSON-LD <a>active context</a> in a way that
+          could compromise security. It is advised that any application that
+          depends on a remote context for mission critical purposes vet and
+          cache the remote context before allowing the system to use it.</p>
+        <p>Given that JSON-LD allows the substitution of long IRIs with short terms,
+          JSON-LD documents may expand considerably when processed and, in the worst case,
+          the resulting data might consume all of the recipient's resources. Applications
+          should treat any data with due skepticism.</p>
+        <p>As JSON-LD places no limits on the IRI schemes that may be used,
+          and vocabulary-relative IRIs use string concatenation rather than
+          IRI resolution, it is possible to construct IRIs that may be
+          used maliciously, if dereferenced.</p>
+      </dd>
+
+      <!-- 
+      Person (including contact information) to contact for further
+      information.
+      -->
+      <dt>Contact</dt>
+      <dd>
+        Ivan Herman &lt;ivan@w3.org&gt;
+      </dd>
```

Comment:
@pchampin, either you should be the contact, or someone more permanent at W3C.

```suggestion
      <dd>Pierre-Antoine Champin &lt;pierre-antoine@w3.org&gt;</dd>
```

Reply:
I will merge with confirmation from @pchampin

Reply:
I think it makes sense to put my contact details here.